src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+  const sum = days * rentCost;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+
+  if (days > 6) {
+    return sum - bigDiscount;
+  } else if (days > 2) {
+    return sum - smallDiscount;
+  }
```

Comment:
Read checklist item 3 again.
If you perform return, code doesn't go further anyway.